tests/ui/components/filters/sortoptionscomponent.js

```diff
@@ -92,23 +91,25 @@ describe('sort options component', () => {
   it('renders correctly for default values', () => {
     const component = COMPONENT_MANAGER.create('SortOptions', defaultConfig);
     expect(component.selectedOptionIndex).toEqual(0);
+    component.handleSearchResults(); // The component will only render if there are search results
```

Comment:
I think this is a breaking change, previously sort options would render even before search results were loaded. We also want to avoid directly calling methods on the component if we can help it

Reply:
I was thinking that not rendering before a search (when there are no results) might better alight with the spec, but if that's not what we want I can easily render it by default

Reply:
which spec?